contracts/treasury/package.json

```
@@ -1,6 +1,6 @@
 {
     "name": "@0x/contracts-treasury",
-    "version": "1.4.37",
+    "version": "1.4.36",
```

This version was published 7 days ago, you may need to rebase off `development`? 
```suggestion
    "version": "1.4.37",
```